nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/query/NxqlQueryConverter.java

```diff
@@ -97,20 +97,14 @@
  * visitor to build the ES request.
  */
 public final class NxqlQueryConverter {
+    protected static final int MORE_LIKE_THIS_MIN_TERM_FREQ = 1;
+    protected static final int MORE_LIKE_THIS_MIN_DOC_FREQ = 3;
+    protected static final int MORE_LIKE_THIS_MAX_QUERY_TERMS = 12;
```

Comment:
check again your indentation configuration, you should have emtpy lines

Reply:
Oh, i am not sure what happened there when i was stashing/unstashing. I will revert that.